docs/reference/go-gno-compatibility.md

```diff
@@ -34,6 +34,11 @@ id: go-gno-compatibility
 
 Generics are currently not implemented.
 
+Note that gno does not support shadowing of native types. 
+While the following native typecasting assignment would work in go, this is not supported in gno.
+
+`rune := rune('a')`
+
```

Comment:
```suggestion
Note that gno does not support shadowing of builtin types. This means that effectively, identifiers like `rune`, `int`, `string` and `byte` are to be considered reserved, and the following will fail:

```go
rune := rune('a')
\```

```

Added the slash so that it formats as a suggestion, but should commit without it. I would avoid `native` as it is confusing (ie. is it gonative? no, it's a builtin identifier)

Reply:
These are referred to as native types directly below this change.  Should we change this to `builtin` as well?

Reply:
yes please!